pytrip/cube.py

```diff
@@ -478,7 +478,7 @@ def create_dicom_base(self):
         ds.StudyID = ''  # Study ID tag 0x0020,0x0010
         ds.ReferringPhysiciansName = ''  # Referring Physician's Name tag 0x0008,0x0090
         ds.PositionReferenceIndicator = ''  # Position Reference Indicator tag 0x0020,0x1040
-        ds.SeriesNumber = ''  # SeriesNumber tag 0x0020,0x0011
+        ds.SeriesNumber = 1  # SeriesNumber tag 0x0020,0x0011
```

Comment:
SeriesNumber is also `Integer String` (which is a string), not `Signed Long` (which is a number), so I propose to use:
`ds.SeriesNumber = '1'`

to avoid magical conversions done by pydicom.